### PR TITLE
Deleted getDefaultLanguage from client

### PR DESCRIPTION
**Pull request by sgiaccio**

### CODE REVIEW

File: webapp/commonComponents/expression/expressionEditorPopup.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ import * as SurveyState from '../../survey/surveyState'
 import * as ExpressionVariables from './expressionVariables'
 import * as ExpressionParser from './expressionParser'
 
+import appContext from '../../app/appContext'
```
Comment (minotogna) [inline webapp/commonComponents/expression/expressionEditorPopup.js]: please let's use uppercase convention as usual

File: common/i18n/resources/en.js [hunk 2]
```diff
@@ -246,10 +247,10 @@ module.exports = {
     TaxonomyImportJob: 'Taxonomy Import',
   },
 
-  formErrors: {
+  errors: {
```
Comment (minotogna) [inline common/i18n/resources/en.js]: let's keep separate formErrors to systemErrors (as agreed)

File: webapp/commonComponents/expression/expressionEditorPopup.js (hunk 1 above)
Comment (sgiaccio) [reply webapp/commonComponents/expression/expressionEditorPopup.js]: Done

File: common/i18n/resources/en.js (hunk 2 above)
Comment (sgiaccio) [reply common/i18n/resources/en.js]: Ok

**Pull request merged**